web/src/beta/lib/core/engines/Cesium/Feature/Tileset/hooks.ts

```diff
@@ -431,13 +438,6 @@ export const useHooks = ({
         return;
       }
 
-      try {
```

Comment:
Maybe we need to test the clipping box as well.

Reply:
I've tested clipping box using view2.0 project, seems working well.